### ISSUE TITLE
bug: extension.stateObj.enable is not a function

### ISSUE THREAD

**Issue by 0xMRTT** [oae/gnome-shell-pano#2]
I just installed from GNOME Extensions and getting an error:

```
extension.stateObj.enable is not a function
```

**Comment by oae** [comment]
Did you install the `libgda` dependency?
________________________________
From: 0xMRTT ***@***.***>
Sent: Friday, August 26, 2022 4:04:13 PM
To: oae/gnome-shell-pano ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [oae/gnome-shell-pano] bug: extension.stateObj.enable is not a function (Issue #2)


I just installed from GNOME Extensions and getting an error:

extension.stateObj.enable is not a function

**Comment by 0xMRTT** [comment]
Yes

**Comment by 0xMRTT** [comment]
Fedora 37

**Comment by oae** [comment]
Seems like Fedora 37 uses gnome-shell 43. I only tested this extension on gnome-shell 42. I will be adding support for gnome-shell 43 soon.

**Comment by RushingAlien** [comment]
I'm on Arch, GNOME 42, i also encounter this error and cannot enable the extension

**Comment by PoorPocketsMcNewHold** [comment]
Happening on Fedora 36 with gnome-shell 42.3.1-2. Nothing logged about the extension in the logs.

**Comment by RushingAlien** [comment]
```
Agu 26 22:07:55 komputer-raambm gnome-shell[1335]: JS ERROR: Extension pano@elhan.io: TypeError: extension.stateObj.enable is not a function
                                                   _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:182:32
                                                   loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:383:26
                                                   _onInstallButtonPressed@resource:///org/gnome/shell/ui/extensionDownloader.js:267:35
                                                   Async*addButton/<@resource:///org/gnome/shell/ui/dialog.js:132:41
```

**Comment by oae** [comment]
I will add more logs with the new version for easy troubleshooting.

**Comment by oae** [comment]
Can you try getting the logs with this release: https://github.com/oae/gnome-shell-pano/releases/tag/v3

**Comment by TJonCooper** [comment]
Also have this error on Ubuntu 22.04 - shell-42.

**Comment by ChinaskiJr** [comment]
Hello ! 
Same here with PopOS! 22.04 (Gnome 42)

**Comment by oae** [comment]
The new version is approved. Can you check and see if a more specific error is logged?

**Comment by TJonCooper** [comment]
Fixed using the new version (Ubuntu 22.04). Thanks!

**Comment by ChinaskiJr** [comment]
The new version fixed it also on my side :) (PopOS! 22.04). Thank you for your amazing work !

**Comment by oae** [comment]
I am closing this issue then. If you encounter a similar issue again, feel free to open a new one.
